index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio by Abenezer Samuel</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <h1 class="logo">Abenezer</h1>
+      <img src="./image/menu.png" alt="menu icon image" />
+    </header>
+    <section class="headline">
+      <h2 class="Primay-text">I'm Abenezer.<br> Glad to see <br>you!</h2>
```

Comment:
- [ ] [REQUIRED] Good job so far! The main title (e.g., "I'm Abenezer ...") should be a `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. The main title (e.g., "I'm Abenezer ...") is a h1 tag.